raml2markdown/src/calendar-api/includes/examples/calendar.json

```diff
@@ -1,6 +1,6 @@
 {
-  "@context": "https://standards.lifeengine.io/v1/Context/Calendar",
-  "@type": "Event",
+  "@context": "https://standards.lifeengine.io/v1/Context/Identity/Calendar/",
```

Comment:
This is still `/Context/Calendar` in code. But the thing is, calendars are not really used anywhere at the moment... We need to update this context in code.

Reply:
I removed `/Identity` part of URL, so it is `https://standards.lifeengine.io/v1/Context/Calendar/` now. Trailing slash is required for standard, so I left it there. I'll create followup task to add `Calendar` and `Message` to standard since they are nto yet there. After that it will be clear what exact URL it will be